modules/rosa-troubleshooting-deployment.adoc

```diff
@@ -40,3 +40,21 @@ $ rosa init --delete-stack
 ----
 $ rosa init
 ----
+
+== Elastic load balancer (ELB) `AccessDenied` error
+
+In AWS accounts that have never created a load balancer, it is possible that the service role for the Elastic load balancer (ELB) might not exist yet. You may receive the following error:
```

Comment:
```suggestion
If you have not created a load balancer in your AWS account, it is possible that the service role for the elastic load balancer (ELB) might not exist. You may receive the following error:
```

Reply:
Suggest flipping some phrases around in the beginning. Also wondering if "elastic" needs to be capitalized in the sentence? When I look at other AWS doc, it seems like it isn't and sometimes seems like it is, like [here](https://docs.openshift.com/container-platform/4.9/installing/installing_aws/installing-aws-account.html) when I ctrl+f, in the table there's capitalized and then elsewhere in a note it isn't.